espanol/angeles/angeles.md

```diff
@@ -6,4 +6,4 @@ han brindado. Betito se llenó de fuerzas también y animó a todos los guerrero
 
 La voz de un ángel te llama, pero estas distraído con la tecnología!
 
-El progreso no mide tiempo.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.